### PR TITLE
Low Priority Format

### PR DESCRIPTION
**Pull request by joseph-cooke**

### CODE REVIEW

File: docs/source/hardware/low_priority.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+(low-priority)=
+
+# Low Priority Partition
+
+## Introduction
+
+`lowpriority` is a new partition on our high performance computing (HPC) cluster, Discovery which allows the research community to use private resources on the HPC cluster when they are idle. This is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. This new partition has hardware that is not otherwise available to the general research community and, in time, could double the resources available to NURC users.
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority.md:7]: Probably remove "Discovery" here and just go with cluster.

File: docs/source/hardware/low_priority.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## When to use the Low Priority Partition
+
+From most to least-recommended scenarios:
+
+1. Code that can be checkpointed (Please see {ref}`checkpoint-jobs`)
+1. Jobs that fit on a single node 
+1. Jobs that require multiple nodes (e.g., MPI - please see {ref}`using-mpi`)
+1. When waiting for your jobs to get started is too hard
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority.md:16]: Maybe rephrase to: When facing longer than average waits times in the other partitions

File: docs/source/hardware/low_priority_faqs.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+::::{dropdown} 2. What is a "low priority" job?
+A job submitted to the `lowpriority` partition that comes from a user **who is NOT** a member of the private partition on which the job is scheduled to run.
+::::
+
+::::{dropdown} 3. What is a “high priority” job?
+A job submitted to a private partition by a user **who is** a member of that private partition, on which a low priority job is running.
+::::
+
+::::{dropdown} 4. How does SLURM handle low priority jobs?
+In SLURM, jobs are submitted to partitions. SLURM is configured such that jobs submitted to the `lowpriority`partition are considered for execution on private partitions only when the requested job resources are idle.
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority_faqs.md:23]: This whole section looks good! I just had one question: Does the node state have to be "idle" for a job to be run? Or is it that the node has to be not in use by a member of the private partition for outside members jobs to be run?

For example, say a node is being used by low priority by a user who isn't part of the private partition, the node is no longer idle, can another job (from a none private partition member) be run on the same node (if allowable with the requested resources)?

File: docs/source/hardware/low_priority_faqs.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+::::
+
+::::{dropdown} 2. What is the understanding between RC and private partition owners?
+The following understanding applies to all individual private partition owners who purchased servers hosted at MGHPCC since 2019:
+
+RC racks, installs and maintains privately owned servers; RC purchases the equipment needed to connect these servers to the infiniband network fabric; RC pays for power and cooling to operate the servers during their entire lifetime; In return, private partition owners make their servers available to the `lowpriority` partition.
+::::
+
+::::{dropdown} 3. Are there policies or other formal documents available?
+RC is currently drafting a memorandum of understanding (MoU) that will be distributed to all private partitions owners later in the Fall semester 2022. This document will formalize the understanding that was stated to all private partitions owners at the time of purchase, since 2019. Moreover, RC is working on a detailed set of documentation, service level objectives and training sessions that will be available later in the Fall 2022 semester, once the service is ready to be released.
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority_faqs.md:146]: Should this say: fall 2023? Or has it happened and needs to be referred to as a past event?

Comment (SerenaCaplins) [reply docs/source/hardware/low_priority_faqs.md:146]: For the message to PIs that is. I think the second fall 2022 should be fall 2023

File: docs/source/hardware/low_priority_faqs.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+::::{dropdown} 2. What is the timeline of the testing phase?
+The `lowpriority` partition will be available for testing after the October maintenance window (October 4, 2022). Only groups who opt in will have access to the `lowpriority` partition during the testing phase and only hardware owned by those groups will be part of the test partition.
+::::
+
+::::{dropdown} 3. How can I participate in the testing phase?
+Let us know by sending an email to [rchelp@northeastern.edu](mailto:rchelp%40northeastern.edu) with the subject “lowpriority partition”. Members of your group will get first access to all newly opened resources for the duration of the testing phase. Only hardware owned by PIs who opt-in will be part of the testing phase.
+::::
+
+::::{dropdown} 4. When will the `lowpriority` partition be officially released?
+Once testing, configuration, and documentation are complete, we will schedule an additional rollout of the `lowpriority` partition and provide access to all users of the HPC cluster. We anticipate this to be in **late 2022/early 2023**.
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority_faqs.md:180]: Change the year here to: 2023/early 2024

File: docs/source/hardware/low_priority_faqs.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+
+## Miscellaneous
+
+::::{dropdown} 1.Where can I learn more about the `lowpriority` partition?
+RC is working on a {ref}`low-priority`, service level objectives, and training sessions that will be available later in the Fall 2022 semester, once the service is ready to be released.
```
Comment (SerenaCaplins) [inline docs/source/hardware/low_priority_faqs.md:192]: Are the training sessions available? Otherwise should be Fall 2023

**Review by SerenaCaplins — APPROVED** [review]
Everything looks good to me. I had a few comments and suggested changes and one question. Thank you!

File: docs/source/hardware/low_priority_faqs.md (hunk 3 above)
Comment (joseph-cooke) [reply docs/source/hardware/low_priority_faqs.md:23]: From my experience, idle is just available resources and not the actual IDLE NODELIST reason. From what I remember, multiple users can all use a single node as long as there are available resources too. Even people who can normally run on the private hardware can coexist with lowpriority users but if the private hardware group requires more resources than what is available that is when jobs start getting cancelled for the low priority users. 

Greg would definitely be able to answer this in more definitely phrasing but this is what I recall from using a lowpriority partition previously but I was on the other end and didn't have complete visibility.

File: docs/source/hardware/low_priority_faqs.md (hunk 4 above)
Comment (joseph-cooke) [reply docs/source/hardware/low_priority_faqs.md:146]: I agree that we need to change these dates. Not sure if this has been distributed yet or still needs to and such. Thank you for catching this because this definitely needs to be changed. Thank you!!

File: docs/source/hardware/low_priority_faqs.md (hunk 5 above)
Comment (joseph-cooke) [reply docs/source/hardware/low_priority_faqs.md:180]: Same as above. We need to formalize when we want to roll this out and then we can adjust appropriately.

File: docs/source/hardware/low_priority_faqs.md (hunk 6 above)
Comment (joseph-cooke) [reply docs/source/hardware/low_priority_faqs.md:192]: Same as above. We need to formalize when we want to roll this out and then we can adjust appropriately. I don't think we ever recorded any training sessions either.
